doc/source/basics.rst

```diff
@@ -793,6 +801,76 @@ This is equivalent to the following
    result
    result.loc[:,:,'ItemA']
 
+.. _basics.assigning:
+
+Assigning New Columns
+---------------------
```

Comment:
I am also not really sure about the best place to put this in the docs, but I would certainly add something about that the 'normal' non-method way to assign new columns is just using `df['new_col'] = ...`. Because, given the current title, it could seem this is 'the' way to assign new columns.